spec/migrations/20170406135022_rename_waiting_to_start_state_to_queuedin_miq_task_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_migration
+
+describe RenameWaitingToStartStateToQueuedinMiqTask do
+  let(:task_stub) { migration_stub(:MiqTask) }
+  let(:task_name) { "Hello Test Task" }
+
+  migration_context :up do
+    it "updates 'state' attribute on 'miq_tasks' table from 'Waiting_to_start' to 'Queued'" do
+      task_stub.create!(:name => task_name, :state => "Waiting_to_start")
+      migrate
+      expect(MiqTask.find_by(:name => task_name).state).to eq MiqTask::STATE_QUEUED
```

Comment:
Also, you can't use the model for the constant reference, but you can copy it into the stub and use that.